spec/index.bs

```diff
@@ -2771,7 +2794,12 @@ A {{GPUBindGroup}} object has the following internal slots:
                                     - |resource|'s texture's {{GPUTextureDescriptor/usage}} includes
                                             {{GPUTextureUsage/SAMPLED}}.
 
-                                - If |layoutBinding|.{{GPUBindGroupLayoutEntry/type}} is {{GPUBindingType/"sampled-texture"}}:
+                                - If |layoutBinding|.{{GPUBindGroupLayoutEntry/type}} is {{GPUBindingType/"filtered-texture"}}:
+                                    - |resource|'s texture's {{GPUTextureDescriptor/format}} must have a "Filter" capability,
+                                        as described in {#plain-color-formats}.
```

Comment:
This should point to #texture-format-caps, to include all of the formats, not just the plain color formats. Also I think this is the correct syntax:
```suggestion
                                        as described in [[#texture-format-caps]].
```